AODN/AODN-WAVE-NRT/ARDC_API_NRT/ardc_nrt/lib/omc/api.py

```diff
@@ -115,14 +115,16 @@ def get_source_id_query(self, query=None):
         # see https://stackoverflow.com/questions/23267409/how-to-implement-retry-mechanism-into-python-requests-library
         retries = Retry(total=5, backoff_factor=1, status_forcelist=[502, 503, 504])
         self.session.mount('https://', HTTPAdapter(max_retries=retries))
-        res = self.session.get(url_request, headers={'Authorization': 'Bearer ' + self.access_token, 'User-Agent': 'UTAS'})
-
-        if res.status_code != 200:
-            if res.status_code == 504:
-                self.logger.error('API Gateway timeout error 504. {error}'.format(error=res))
-            else:
-                self.logger.error(error=res)
-            return
+        try:
+            res = self.session.get(url_request, headers={'Authorization': 'Bearer ' + self.access_token, 'User-Agent': 'UTAS'})
```

Comment:
was it really failing at this line? or was the status code different?

Reply:
it was failing at the line where the error was logged